docs/sources/send-data/otel/native_otlp_vs_loki_exporter.md

```diff
@@ -0,0 +1,93 @@
+---
+title: How is native OTLP endpoint different from Loki Exporter
+menuTitle:  Native OTLP endpoint vs Loki Exporter
+description: Native OTLP endpoint vs Loki Exporter
+weight:  251
+---
+
+# Introduction
+
+OTel is quickly becoming an industry standard with increasing adoption. Prior to the Loki 3.0 release, there was no native support for ingesting OTel logs to Loki, which led to creation of [LokiExporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/lokiexporter/README.md). While the LokiExporter got the job done of ingesting OTel logs to Loki, it did not provide a native user experience, and the querying experience was not as optimal. As part of our effort to improve user experience with OTel, we added native OTel log ingestion support to Loki with 3.0 release.
+
+# What has changed?
+
+## Formatting of logs
+
+[LokiExporter's README](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/lokiexporter/README.md) explains in detail the formatting of OTel logs ingested via LokiExporter, while Loki’s native OTel log ingestion endpoint is described in detail in its documentation. Here is a summary of how the logs are formatted between the two:
+
+### LokiExporter
+
+- Index Labels:
+  - It supports label control via hints set by the OTel client/collector.
+  - Default labels:
+    - job=service.namespace/service.name 
+    - instance=service.instance.id 
+    - exporter=OTLP 
+    - level=severity
+- Log Body: Encodes log records and attributes in json(default) or logfmt format.
+
+### Loki’s native OTel log ingestion endpoint:
+
+- Index Labels:
+  - It supports label control via per-tenant OTLP configuration in Loki. 
+  - By default, it picks some resource attributes as index labels.
+- LogLine: Stringified LogRecord.Body.
+- Structured Metadata: Anything not stored in Index labels and LogLine gets stored as Structured Metadata.
+
+### Sample Log
+
+Let us take the following sample OTel log and see how it would look after ingestion in both formats with the default configuration:
+- Resource Attributes:
+  - service.name: “auth”
+  - service.namespace: “dev”
+  - service.kind: “app”
+- Log Record:
+  - Timestamp: 1715247552000000000
+  - Body: “user logged in”
+  - Severity Text: “INFO”
+  - Attributes:
+    - email: “foo@bar.com”
+
+#### Ingested with LokiExporter:
```

Comment:
This shouldn't be a heading, it's still part of the Sample Log section.

Reply:
I have updated the document so that it does not have it as a heading and made it look bold to stand out.